Lib/test/test_launcher.py

```diff
@@ -748,6 +748,9 @@ def test_shebang_command_in_venv(self):
         with self.script(f'#! /usr/bin/env {stem} arg1') as script:
             data = self.run_py([script], expect_returncode=103)
 
+        if not is_installed("3"):
```

Comment:
The function checks for specific registry keys, so you'll need a `3.x` or `3.x-32`/`3.x-arm64` for this to ever return True.